src/main/java/frc/robot/ProfileExternalPIDController.java

```diff
@@ -0,0 +1,144 @@
+package frc.robot;
+import edu.wpi.first.math.trajectory.TrapezoidProfile;
+
+public class ProfileExternalPIDController {
```

Comment:
This is the old ProfiledExternalPIDController. WPILib made updates which are reflected in the file here: [https://github.com/FRC2706/2023-2706-Fall-Poseidon/blob/Erik-ArmSim/src/main/java/frc/lib/lib2706/ProfiledExternalPIDController.java](https://github.com/FRC2706/2023-2706-Fall-Poseidon/blob/Erik-ArmSim/src/main/java/frc/lib/lib2706/ProfiledExternalPIDController.java)